pages/posts/stability.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Towards a stable JSON Schema"
+date: 2022-10-08
+type: Update
+tags:
+  - Standardization
+cover: /img/posts/2022/stability/arches.webp
+authors:
+  - name: Jason Desrosiers
+    photo: /img/avatars/jasondesrosiers.jpeg
+    link: https://twitter.com/jasondesrosiers
+    byline: JSON Schema Specification Contributor
+---
+
+About this time last year, I hosted a discussion at the API Specification
+Conference about the future of JSON Schema. The most popular topic of discussion
+was, when is JSON Schema going to be "done". Of course we've heard that question
+a lot. The question stems from the "draft" label we put on your releases. The
+term "draft" has led to quite a bit of confusion for our community over the
+years, so let's take a moment to understand where it comes from.
+
+## What does "draft" mean?
+
+JSON Schema has been loosely following the IETF process for a standards track
+IETF RFC. Generally what happens in this process is that a person or
+group of people write an Internet Draft (I-D). This is often referred to as an
+"individual draft". At this point, the I-D is not accepted or supported by the
+IETF. It's effectively a proposal at this point. It's entirely individuals
+working on their own with the hope of some day getting their I-D accepted to an
+IETF Working Group (WG). Once accepted by a WG, the I-D has access to IETF
+resources and support and is on it's way to becoming an RFC. The I-D will
+typically go through several revisions and will eventually be accepted as a
+standards track RFC.
+
+JSON Schema has always been at the "individual draft" stage of this process. It
+was never on a standards track. There have been two major reasons why JSON
+Schema hasn't progress further than this in the IETF process. The first reason
+is that JSON Schema keeps evolving. Every year we get feedback from our
+community and we make changes or add features to respond to that feedback. If
+JSON Schema were to be standardized, responding to our community would be a
+difficult and slow processes. We believe that continuing to respond to the needs
+of the community is crucial for JSON Schema and we want to be able to continue
+doing that.
+
+The second reason we haven't pursued a standards track RFC is that the
+circumstances JSON Schema is subject to is not really compatible with the normal
+I-D process. It would be hard to jump into a WG and have to conform to a process
+that doesn't really fit the realities of our situation.
+
+Normally, each time a draft is released, it replaces the previous draft. That
+makes sense when you are developing a spec that people don't rely on yet, but
+JSON Schema is widely used in almost every corner of the web. Every time we
+release a new draft, that draft will be used and depended on in production
+systems. It can't simply be considered obsolete. Implementations need to
+continue to support those older drafts including backward incompatible changes.
```

Comment:
I think the second paragraph (in this selection) is intended to explain the first, but it's not quite clear.  On my initial read, my impression was, "JSON Schema isn't compatible with the I-D process.  Moving on...".  It seemed like you made the claim without backing it up.

Maybe if these were just the same paragraph?

Reply:
I was concerned that that transition was too awkward. Thanks for confirming. I'll see what I can do.